backend/serverless.yml

```diff
@@ -1,19 +1,25 @@
 service: backend
 frameworkVersion: '3'
 
-useDotenv: true
 plugins:
   - serverless-plugin-typescript
+  - serverless-dotenv-plugin
```

Comment:
Do take note that using `serverless-dotenv-plugin` also deploys the envars to the lambda. Not sure if that's what you intended

Reply:
Yup, all the .env files are for non-secret configuration, secrets will be handled by other methods (like Github Secrets)